pkg/controller/user-cluster-controller-manager/resources/resources/csi/migration/secret.go

```diff
@@ -24,38 +24,37 @@ import (
 	"k8c.io/reconciler/pkg/reconciling"
 
 	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	certutil "k8s.io/client-go/util/cert"
 )
 
-var webhookConfig = fmt.Sprintf(`
+const webhookConfig = `
 [WebHookConfig]
-port = "%d"
-cert-file = "/etc/webhook/cert.pem"
-key-file = "/etc/webhook/key.pem"
-`, resources.CSIMigrationWebhookPort)
+port = "8443"
+cert-file = "/run/secrets/tls/cert.pem"
+key-file = "/run/secrets/tls/key.pem"
+`
 
 func TLSServingCertificateReconciler(ca *triple.KeyPair) reconciling.NamedSecretReconcilerFactory {
```

Comment:
Nit: can we name this function something like `VsphereTLSServingCertificateReconciler` to make it clear it's supposed to be only used for vSphere?